File: in_Java/1389 Create Target Array in the Given Order.java
class Solution {
    public int[] createTargetArray(int[] nums, int[] index) {
        List<Integer> L = new ArrayList<>();
        for(int i = 0; i < nums.length; i++){
            L.add(index[i], nums[i]);
        }
        int[] res = new int[nums.length];
        for(int i = 0; i < res.length; i++) res[i] = L.get(i);
        return res;
    }
}

File: in_Python/1389 Create Target Array in the Given Order.py
class Solution(object):
    def createTargetArray(self, nums, index):
        """
        :type nums: List[int]
        :type index: List[int]
        :rtype: List[int]
        """
        res = []
        for i in range(len(nums)):
            n = nums[i]
            idx = index[i]
            res.insert(idx,n)
        return res